src/test/resources/features/ifs/InterestRateChanges.feature

```diff
@@ -0,0 +1,131 @@
+    And the debt item has payment history
+      | amountPaid | dateOfPayment |
+      | 100000     | 2021-01-20    |
+      | 100000     | 2021-03-10    |
+    When the debt item is sent to the ifs service
+    Then the ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | interestDueCallTotal | unpaidAmountTotal | totalAmountIntTotal | amountOnIntDueTotal |
+      | 49                   | 32944                | 700000            | 732944              | 700000              |
+    And the 1st debt summary will contain
+      | interestDueDailyAccrual | interestDueDebtTotal | unpaidAmountDebt | totalAmountIntDebt | amountOnIntDueDebt |
+      | 28                      | 16927                | 400000           | 416927             | 400000             |
+    And the 1st debt summary will have calculation windows
+      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | interestDueWindow | unpaidAmountWindow | amountOnIntDueWindow |
+      | 2020-01-01 | 2020-03-29 | 89           | 3.25         | 44                      | 3962              | 500000             | 500000               |
+      | 2020-03-30 | 2020-04-06 | 8            | 2.75         | 37                      | 301               | 500000             | 500000               |
+      | 2020-04-07 | 2021-03-14 | 342          | 2.6          | 35                      | 12180             | 500000             | 500000               |
+      | 2021-03-15 | 2021-03-31 | 17           | 2.6          | 28                      | 484               | 400000             | 400000               |
+    And the 2nd debt summary will contain
+      | interestDueDailyAccrual | interestDueDebtTotal | unpaidAmountDebt | totalAmountIntDebt | amountOnIntDueDebt |
+      | 21                      | 16017                | 300000           | 316017             | 300000             |
+    And the 2nd debt summary will have calculation windows
+      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | interestDueWindow | unpaidAmountWindow | amountOnIntDueWindow |
+      | 2020-01-16 | 2020-03-29 | 74           | 3.25         | 44                      | 3294              | 500000             | 500000               |
+      | 2020-03-30 | 2020-04-06 | 8            | 2.75         | 37                      | 301               | 500000             | 500000               |
+      | 2020-04-07 | 2021-01-19 | 288          | 2.6          | 35                      | 10257             | 500000             | 500000               |
+      | 2021-01-20 | 2021-03-09 | 49           | 2.6          | 28                      | 1396              | 400000             | 400000               |
+      | 2021-03-10 | 2021-04-14 | 36           | 2.6          | 21                      | 769               | 300000             | 300000               |
+
+  Scenario: Interest rate changes from 2.75% to 2.6% - dateCalculationTo before interestStartDate

```

Comment:
Interest rate used in the scenario is 3.25 and dateCalculationTo is after interestStartDate